src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -29,13 +29,8 @@ export const Container = styled.div<CheckboxStateProps>`
   border-radius: 100%;
   color: ${colors.gray[300]};
   transition: background ${transitions.timings.loading} ${transitions.easing};
-  ${({ isFocused, selected, disabled }) =>
-    isFocused && !disabled && `background: ${selected ? colors.blue[900] : colors.gray[800]};`}
   &:hover {
-    ${({ selected, disabled, error }) => [
-      !disabled && `background: ${selected ? colors.blue[900] : colors.gray[800]};`,
-      error && `background: ${shade(0.7, colors.error)};`,
-    ]}
+    ${({ disabled }) => [!disabled && `background: var(--hoverColor)`]}
```

Comment:
I don't think we need to declare that variable and use it if it's used only here